plugins/org.python.pydev.shared_core/src/org/python/pydev/shared_core/actions/LineCommentAction.java

```diff
@@ -59,9 +58,25 @@ public FastStringBuffer commentLines(String selectedText) {
         FastStringBuffer lineBuf = new FastStringBuffer();
         boolean addSpacesInStartComment;
 
+        int lowestIndent = -1;
+        if (LineCommentOption.ADD_COMMENTS_INDENT.equals(addCommentsOption)) {
+            for (String line : ret) {
+                int indent = TextSelectionUtils.getIndentationFromLine(line).length();
```

Comment:
Instead of calling `TextSelectionUtils.getIndentationFromLine(line).length()`, you could call `TextSelectionUtils.getFirstCharPosition(line)`.

That way you don't need to create a temporary string variable to get its length...